### DISCUSSION TITLE
Postal has stopped sending emails,

### DISCUSSION THREAD

**Discussion by fav22** [postalserver/postal#1826] [Help with using Postal]
## Describe the bug

Hello, I have a problem with postal. He stopped sending emails from my email client, all messages have been in the queue for almost 2 hours and it's still stuck there, I've already tried to restart postal and my server but nothing has changed.

## Environment details 

 - OS:  ubuntu
 - Browser  chrome
 - Version : 20.4
 - Type  desktop

**Comment by willpower232** [comment]
Are you able to have a look at the queued_messages table? This sounds like https://github.com/postalserver/postal/issues/287#issuecomment-331081922

**Reply by fav22** [reply]
from database you mean ? No i'm not able

**Reply by fav22** [reply]
can you tell me how to to have access to it from ssh please , i'm using ubuntu 20.4

**Reply by willpower232** [reply]
You could use an SSH tunnel, it depends on how you have your database set up.

**Reply by fav22** [reply]
after checking `service rabbitmq-server status` i got this , what does it mean please


`● rabbitmq-server.service - RabbitMQ Messaging Server
     Loaded: loaded (/lib/systemd/system/rabbitmq-server.service; enabled; vendor preset: enabled)
     Active: activating (auto-restart) (Result: exit-code) since Mon 2022-01-24 13:02:09 CET; 183ms ago
    Process: 235511 ExecStart=/usr/sbin/rabbitmq-server (code=exited, status=1/FAILURE)
   Main PID: 235511 (code=exited, status=1/FAILURE)
`

**Reply by willpower232** [reply]
You'll have to look at log files for it to see what its problem is.

**Reply by fav22** [reply]
log of which service please ?

**Reply by willpower232** [reply]
You'll have to look at the log files of RabbitMQ to see why it did not start.
